api/src/main/java/io/strimzi/api/kafka/model/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -138,6 +140,28 @@ public void setUserNameClaim(String userNameClaim) {
         this.userNameClaim = userNameClaim;
     }
 
+    @Description("Configure if access token type check should be skipped. Set this to `true` if authorization server " +
+            "does not include 'typ' claim in JWT token. Defaults to `false`.")
```

Comment:
I think typ is not a typo, but I just want to check.

Reply:
Correct. Not a typo.